odoo/addons/test_translation_export/tests/test_static.py

```diff
@@ -0,0 +1,10 @@
+from odoo.tools.translate import TranslationModuleReader
+from odoo.tests import common, tagged
+
+
+@tagged('post_install', '-at_install')
+class TestExport(common.TransactionCase):
+
+    def test_export_translatable_resources(self):
+        """Read files of installed modules and export translatable terms"""
+        TranslationModuleReader(self.env.cr, export_from_db=False, strict_mode=True)
```

Comment:
We may also want to mute_logger odoo.addons.base.models.ir_module to avoid spamming the log